src/test/java/ast/sibling/2.out

```diff
@@ -0,0 +1,3 @@
+A
+    B
+    C
```

Comment:
Why do some of the tests have an `.out` and others don't? I'm talking about `2.out` in `/sibling` and `2.out` in `/child`.

Reply:
Because only write .out files when the format is different than the input, for example, these two cases.

Reply:
Now it makes sense, thank you!